ScienceJournal/UI/ObserveViewController.swift

```diff
@@ -176,6 +176,8 @@ open class ObserveViewController: ScienceJournalCollectionViewController, ChartC
     return self.observeDataSource.availableSensorIDs
   }
 
+  @objc dynamic private(set) var unusedSensors: Bool = false
```

Comment:
`hasUnusedSensors` would be more consistent with the property naming guidelines for boolean values.